contracts/src/interfaces/AbstractStakeTable.sol

```diff
@@ -69,6 +69,14 @@ abstract contract AbstractStakeTable {
         EdOnBN254.EdOnBN254Point schnorrVK;
     }
 
+    /// @notice Struct representing a queue
+    /// @param firstAvailableEpoch epoch where the queue is located.
+    /// @param pendingRequests current size of the queue.
+    struct Queue {
+        uint64 firstAvailableEpoch;
+        uint64 pendingRequests;
+    }
```

Comment:
I prefer flatten this. 
struct helps with mental load by introducing abstraction at the cost of slight overhead (in terms of abi encoded bytes).